backend/db/migrate/20250808120019_remove_lawyers_enabled_from_companies.rb

```
@@ -0,0 +1,5 @@
+class RemoveLawyersEnabledFromCompanies < ActiveRecord::Migration[7.2]
```

RuboCop (1.76.1)

[convention] Missing frozen string literal comment.

(Style/FrozenStringLiteralComment)